packages/website/src/bin/handle-config.js

```diff
@@ -50,11 +53,13 @@ const processConfig = (cwd, providedConfig = {}) => {
   );
 
   return {
-    docsList,
-    packagesPaths,
+    // readMePath/customPackageFields should be overridable by config
     readMePath,
```

Comment:
Maybe readMePath shouldn't be overridable, and we should just require people to use `README.md`?